components/BorrowPopup/BorrowPopup.module.css

```
@@ -26,6 +26,11 @@
   text-align: center;
 }
 
+:global(.dark) .popupContent {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)